src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -78,6 +78,10 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <DefineConstants>$(DefineConstants);TARGET_ARM64</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Platform)' == 'loongarch64'">
```

Comment:
Could you please also add `'$(Platform)' == 'loongarch64'` for `Is64Bit` in `src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems`?

Reply:
Thanks for your review.
I will do it right now.